monero/deploy.yaml

```diff
@@ -3,40 +3,36 @@ version: "2.0"
 
 services:
   xmrig:
-    image: andrey01/xmrig:latest
+    image: cryptoandcoffee/akash-xmrig:1
```

Comment:
it'd be good to have the source of this be available.

Reply:
The source is the Dockerfile in the repo.
https://github.com/ovrclk/awesome-akash/blob/xmrig/monero/Dockerfile